docker-images/jenkins/Dockerfile

```diff
@@ -15,19 +15,12 @@
 FROM jenkins/jenkins:lts
 USER root
```

Comment:
@Aaquiff is it mandatory to create and run a container out of the image created with `root` as the default user?

Reply:
@chirangaalwis we need root to install Docker and other packages. We need to switch back to Jenkins user.